core/src/test/java/org/aerogear/mobile/core/unit/configuration/ServiceConfigurationTest.java

```diff
@@ -12,11 +12,11 @@
 
     @Test
     public void testCreateConfig() {
-        ServiceConfiguration config = ServiceConfiguration.newConfiguration().setName("conf-name")
+        ServiceConfiguration config = ServiceConfiguration.newConfiguration().setId("conf-id")
                         .setType("conf-type").setUrl("http://test-uri.feedhenry.org")
                         .addProperty("prop1", "value1").addProperty("prop2", "value2").build();
 
-        Assert.assertEquals("conf-name", config.getName());
+        Assert.assertEquals("conf-id", config.getId());
```

Comment:
Could we have this value in a final string? It is asking for a typo :P

Reply:
I wouldn't create constants for those in the tests :) In case of a typo, the tests fail and you can fix the typo immediately.